content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/800-implicit-to-explicit-conversion.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Converting Implicit many-to-many relation to Explicit many-to-many relation'
+metaTitle: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+metaDescription: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+---
+
```

Comment:
I think you need some component on the top level for the document to be rendered correctly - other existing files should give an idea.